tracardi/process_engine/action/v1/internal/add_empty_profile/plugin.py

```diff
@@ -45,7 +45,9 @@ async def run(self, payload: dict, in_edge=None) -> Result:
                 "Your requested an update of event profile but events may not be updated in debug mode.")
         else:
             self.event.update = True
-        self.execution_graph.set_profiles(profile)
+
```

Comment:
When the execution_graph can be None? What is the usecase?

Reply:
unit tests, we have no graph since we run one plugin and that's it

Reply:
OK then we have to write the test differently because test cannot change the system source. If there was an error in the system regarding the empty execution graph we will never see it.

Reply:
Alright, gonna change it asap

Reply:
done